packages/blaze/template.js

```diff
@@ -431,6 +431,26 @@ Blaze.TemplateInstance.prototype.subscriptionsReady = function () {
   return this._allSubsReady;
 };
 
+
+/**
+ * @summary Accesses other template instances that enclose the current template instance.
+ * @locus Client
+ * @function
+ * @param {Integer} [numLevels] The number of levels beyond the template instance to look. Defaults to 1.
+ */
+Blaze.TemplateInstance.prototype.parentTemplate = function (numLevels) {
+    var view = Blaze.currentView;
+    if (typeof numLevels === "undefined") {
+        numLevels = 1;
+    }
+    while (view) {
+        if (view.name.substring(0, 9) === "Template." && !(numLevels--)) {
```

Comment:
Uhu. This is really bad. Using view name to determine the nature of the view? This is not how it should be done.